storage/streams/src/main/java/io/apicurio/registry/streams/StreamsRegistryStorage.java

```diff
@@ -546,6 +543,33 @@ public void deleteArtifactRule(String artifactId, RuleType rule) throws Artifact
         }
     }
 
+    @Override
+    public VersionSearchResults searchVersions(String artifactId, Integer offset, Integer limit) {
+
+        if (offset == null) {
```

Comment:
Perhaps rather set these defaults in REST, so you don't have to do it for every storage.
And let the storage have primitives.

Reply:
Right, I'll change it for the searchArtifacts operation as well.